python/kserve/kserve/utils/utils.py

```diff
@@ -133,3 +134,31 @@ def to_headers(context: ServicerContext) -> Dict[str, str]:
         headers[metadatum.key] = metadatum.value
 
     return headers
+
+
+def get_predict_input(payload: Union[Dict, InferRequest]):
+    if isinstance(payload, Dict):
+        return payload["inputs"] if "inputs" in payload else payload["instances"]
+    elif isinstance(payload, InferRequest):
+        input = payload.inputs[0]
+        if input.datatype == "MIXED":
```

Comment:
For mixed we need to construct the data frame from multiple inputs.

Reply:
@Suresh-Nakkeran we should get the all the inputs and return pandas framework if data types are different

Reply:
Let's remove the check on line 144 as it is in a separate PR